Sources/ProjectDescription/LaunchArgument.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public struct LaunchArgument: Equatable, Codable {
+    // MARK: - Attributes
+
+    public let name: String
+    public let isEnabled: Bool
+
+    // MARK: - Init
+
+    public init(name: String, isEnabled: Bool) {
```

Comment:
This is a user-facing model and thus documentation of the public elements will be useful when editing the manifests.